### DISCUSSION TITLE
无法打开官网，启动器无法登录账户，无法检查更新，报错 “出现未知错误”

### DISCUSSION THREAD

**Discussion by EmperorPi** [natfrp/wiki#70] [使用问题]
报错窗口内容如下
自动登录失败, 将在稍后重试: 出现未知错误
System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 111.31.111.45:443
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   在 System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 SakuraFrpService.Natfrp.<Request>d__5.MoveNext()

**Comment by fakeboboliu** [comment]
如果您不是极客用户，不喜欢到处改各种系统配置
---
大概率是你的运营商在搞事，请尝试使用 [阿里公共DNS](https://alidns.com/knowledge?type=SETTING_DOCS#user) 排除运营商 DNS 的问题。

如果无效，请使用手机开热点重试。

如果你是极客用户，经常改各种系统配置
---
你是极客用户，你知道自己搞了哪里但我们不知道，我们不想算卦
